docs/building-a-form/creating-a-form-config-file.md

```diff
@@ -18,6 +18,9 @@ This example `form.js` file will get you started with building your form.
   // Prefix string to add to the path for each page.
   urlPrefix: '',
 
+  // Print the form data to the JavaScript console before it's submitted, for debugging.
```

Comment:
> Print the form data to the JavaScript console before it's submitted, for debugging.

Let's swap this: "For debugging, prints the form data to the JavaScript console before it's submitted."